### PR TITLE
Novadev sprint 8

### PR DESCRIPTION
**Pull request by rezakalfane**
- Updated Toolbar component with Toolbar Panels 
- New information icons when in VSE mode
- New Components
- New Extension

### CODE REVIEW

File: amplience-automation/automation-files/schema/schemas/card-list-schema.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "description": "A list of cards",
+    "allOf": [
+        {
+            "$ref": "http://bigcontent.io/cms/schema/v1/core#/definitions/content"
+        }
+    ],
+    "type": "object",
+    "properties": {
+        "header": {
+			"title": "Header",
```
Comment (rs-amp) [inline amplience-automation/automation-files/schema/schemas/card-list-schema.json:14]: Weird indentation.

File: amplience-automation/automation-files/schema/schemas/rich-text-schema.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (rs-amp) [inline amplience-automation/automation-files/schema/schemas/rich-text-schema.json:2]: This file is indented with tabs.

File: amplience-automation/automation-files/schema/schemas/shoppable-image-schema.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
```
Comment (rs-amp) [inline amplience-automation/automation-files/schema/schemas/shoppable-image-schema.json:2]: This file is also indented with tabs.

**Review by neilmistryamplience — APPROVED** [review]
Approved all changed

**Pull request merged**